apps/edxapp/vars/all/main.yml

```diff
@@ -26,6 +26,9 @@ edxapp_i18n_languages:
   - fr
 edxapp_tx_secret_name: "edxapp-tx-{{ edxapp_vault_checksum | default('undefined_edxapp_vault_checksum') }}"
 
+# -- pvc name
+edxapp_pvc_media_name: "edxapp-pvc-media"
```

Comment:
```suggestion
edxapp_v_media_pvc_name: "edxapp-pvc-media"
```

Reply:
Not in this case, there are 2 variables.

First one ``edxapp_pvc_media_name`` is the name of the PV create here https://github.com/openfun/arnold/blob/681dba5705439c4ffb8ab8b14c3d552c94ff5c47/apps/edxapp/templates/volumes/media.yml.j2#L4
It's the only volume with a variable to specified the PVC name. I keep it to avoid breaking change.

The second ``edxapp_v_media_pvc_name`` is to select the PVC to use for the edxapp's media volume.